content/pipeline/configuration.md

```diff
@@ -124,18 +124,19 @@ type: docker
 name: frontend
 
 steps:
-- name: build
+
+* name: build
   image: node
   commands:
-  - npm install
-  - npm test
-{{< / highlight >}}
+  * npm install
```

Comment:
@normal-coder this line, as well as 128 and 132 should start with `-`, `*` is invalid yaml